src/menu/MenuItem.ts

```diff
@@ -0,0 +1,118 @@
+import { assign } from '@dojo/core/lang';
+import { HNode } from '@dojo/widget-core/interfaces';
+import { v } from '@dojo/widget-core/d';
+import ThemeableMixin, { theme, ThemeableProperties } from '@dojo/widget-core/mixins/Themeable';
+import WidgetBase from '@dojo/widget-core/WidgetBase';
+import * as css from './styles/menu.css';
+
+export interface MenuItemProperties extends ThemeableProperties {
+	/**
+	 * Indicates whether the menu is disabled. If true, then the widget will ignore events.
+	 */
+	disabled?: boolean;
+
+	/**
+	 * Applies only when a URL is provided. If `true`, the URL will be opened in a new window.
+	 */
+	external?: boolean;
+
+	/**
+	 * Returns an object of aria properties to apply to the widget's DOM element.
+	 */
+	getAriaProperties?: () => { [key: string]: string; };
```

Comment:
@smhigley may have specifics, but you can check https://www.w3.org/WAI/tutorials/menus/. @smhigley and I have been adding a camel-cased property without the "aria" for each ARIA attribute that needs to be set externally. [Example](https://github.com/dojo/widgets/blob/master/src/textinput/TextInput.ts#L16). We should stay consistent.